docs/source/endpoints/site.md

```diff
@@ -0,0 +1,24 @@
+---
+html_meta:
+  "description": "Site endpoint for Plone REST API"
+  "property=og:title": "Site endpoint for Plone REST API"
+  "property=og:description": "Site endpoint for Plone REST API"
+  "keywords": "Plone, plone.restapi, REST, API, site, navigation root"
+---
+
+# Site
+
+The `@site` endpoint provides site-wide information, such as the site title, logo, and other information, which is useful to offer generic information about the Plone site.
```

Comment:
@erral if I understand correctly the idea is that the information in the "site" endpoint is publicly available all the time, right? It uses the "zope2.View" permission. I guess it would be worth to mention this here.

The example uses a GET request with auth credentials, maybe we can leave them to make this more obvious. What do you think?

 permission=

Reply:
I think it should be `zope2.View` indeed.